platform-sdk/swirlds-merkledb/src/timingSensitive/java/com/swirlds/merkledb/MerkleDbDataSourceTest.java

```
@@ -794,6 +814,15 @@ public static void assertHash(final MerkleDbDataSource dataSource, final long pa
         }
     }
 
+    public static void assertNullHash(final MerkleDbDataSource dataSource, final long path) {
+        try {
+            assertNull(dataSource.loadHash(path));
+        } catch (final Exception e) {
+            e.printStackTrace();
+            fail("Exception should not have been thrown here!");
```

Could we instead use https://junit.org/junit5/docs/5.0.1/api/org/junit/jupiter/api/Assertions.html#fail-java.lang.String-java.lang.Throwable- instead of printing the stack trace ourselves? I think this will simplify the failing tests report.